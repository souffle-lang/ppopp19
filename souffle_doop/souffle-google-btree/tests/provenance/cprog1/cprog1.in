setdepth 8
explain loop(1, 4, 10, 100)
explain loop(2, 4, 36, 100)
exit
