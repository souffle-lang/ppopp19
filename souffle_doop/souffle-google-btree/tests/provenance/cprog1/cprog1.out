Explain is invoked.
Enter command > Depth is now 8
Enter command > loop(0, 0, 10, 100)                                                                                     
----------------(R1)                                                                                    
loop(1, 0, 10, 100)  loop(0, 0, 10, 100)                                                                
-------------------------------------(R2)                                                               
           loop(1, 1, 10, 100)            loop(0, 0, 10, 100)                                           
----------------------------------------------------------(R2)                                          
                     loop(1, 2, 10, 100)                       loop(0, 0, 10, 100)                      
-------------------------------------------------------------------------------(R2)                     
                                loop(1, 3, 10, 100)                                 loop(0, 0, 10, 100) 
----------------------------------------------------------------------------------------------------(R2)
                                          loop(1, 4, 10, 100)                                           
Enter command > subproof loop() subproof loop()  loop(0, 0, 10, 100)                                                                                                                                                                                                    
------------------------------(R4) ----------------(R1)                                                                                                                                                                                                   
       loop(2, 0, 30, 100)         loop(1, 0, 10, 100)   loop(0, 0, 10, 100)                                                                     loop(0, 0, 10, 100)                                                                                      
----------------------------------------------------(R4) ----------------(R1)                                                                    ----------------(R1)                                                                                     
                  loop(2, 0, 31, 100)                    loop(1, 0, 10, 100)   loop(0, 0, 10, 100)                                               loop(1, 0, 10, 100)  loop(0, 0, 10, 100)                                                                 
--------------------------------------------------------------------------(R4) ----------------(R1)                                              -------------------------------------(R2)                                                                
                             loop(2, 0, 32, 100)                               loop(1, 0, 10, 100)   loop(0, 0, 10, 100)                                    loop(1, 1, 10, 100)            loop(0, 0, 10, 100)                                            
------------------------------------------------------------------------------------------------(R4) ----------------(R1)                        ----------------------------------------------------------(R2)                                           
                                        loop(2, 0, 33, 100)                                          loop(1, 0, 10, 100)   loop(0, 0, 10, 100)                        loop(1, 2, 10, 100)                       loop(0, 0, 10, 100)                       
----------------------------------------------------------------------------------------------------------------------(R4) ----------------(R1)  -------------------------------------------------------------------------------(R2)                      
                                                   loop(2, 0, 34, 100)                                                     loop(1, 0, 10, 100)                                   loop(1, 3, 10, 100)                                 loop(0, 0, 10, 100)  
--------------------------------------------------------------------------------------------------------------------------------------------(R4) ----------------------------------------------------------------------------------------------------(R2) 
                                                              loop(2, 0, 35, 100)                                                                                                          loop(1, 4, 10, 100)                                            
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------(R4)
                                                                                                                   loop(2, 4, 36, 100)                                                                                                                    
Enter command > Exiting explain
